posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -254,10 +255,147 @@ More information about Expression Language 5.0 can be found in:
 
 * link:{url-prefix}/docs/latest/reference/feature/expressionLanguage.html[Open Liberty Expression Language feature documentation]
 
+[#concurrency]
+=== Concurrency 3.0 for Jakarta EE 10
```

Comment:
```suggestion
== Jakarta EE 10 Concurrency 3.0
```

Should be same header as the others and should match the format of Expression Language.